language-support/ts/daml-ledger/index.ts

```diff
@@ -602,10 +642,12 @@ class Ledger {
           }
         }
       }
+      lastContractId = contract ? contract.contractId : null
```

Comment:
I know I'm guilty of this myself... But I'm also still learning, particularly through reading code.
```suggestion
      lastContractId = contract !== null ? contract.contractId : null
```